.github/actions/build-single-project/action.yml

```diff
@@ -0,0 +1,106 @@
+name: 'build-single-playground-project'
+description: 'Builds one playground project based on inputs'
+inputs:
+  project:
+    description: "Name of the project (used for artifacts)"
+    required: true
+  project-root:
+    description: "Root folder of the project (used for building)"
+    required: true
+  gradle-cache-password:
+    description: "password for gradle cache"
+  gradle-enterprise-access-key:
+    description: "access key for gradle enterprise"
+  gradle-flags:
```

Comment:
Why is this one not required? We seem to always set it.

Reply:
they may not be available in the caller (actually, the secret is only available in post-submit builds)